mapdata/WarcraftLegacies/UpgradeData/Skin/1160917074.json

```diff
@@ -39,37 +39,37 @@
       "Level": 1,
       "Id": 1835101799,
       "Type": 3,
-      "Value": "Light\u0027s Praise Adept Training"
+      "Value": "Light\u0027s Praising Adept Training"
```

Comment:
Feels like an odd grammar change, what's it about?

Reply:
I'm not sure... I haven't touched that...

Reply:
I can fix it